### DISCUSSION TITLE
How to Upgrade from v1.90.2 to v.199

### DISCUSSION THREAD

**Discussion by pedigor** [immich-app/immich#8313] [Q&A]
Hello everybody!
I never did any upgrade before, I just installed immich on dez/2023 and started to store my home photos. I searched some documentation here and the official website but I still with some doubts.

There is any documentation for upgrading step by step or just the released note of every version?

The original installation that I did was via Portainer, for the upgrade it's the same thing but with the new docker-compose.yml files? 

There are any database scripts that I should run before or after?

Can I just jump to the v.1.99.0 or do I need to upgrade for minor versions first and if it's the case with ones?

Thank you!

**Comment by alextran1502** [comment]
Hello, I would do incremental updates from each release note to make sure you have things configured properly
